### PR TITLE
NBS-4789: [Disk Manager] refactor sre tools

### PR DESCRIPTION
**Pull request by BarkovBG**
need to return []TaskInfo not []string from ListTasksRunning/ListTasksCancelling methods because I want to use them in collect_lister_metrics_task

https://github.com/ydb-platform/nbs/blob/fc1dd1fa9accb81ec0f7c2a5df231817bc56fb90/cloud/tasks/collect_lister_metrics_task.go#L107

### CODE REVIEW

**Review by leftmain — APPROVED** [review]

File: cloud/tasks/tasks_tests/tasks_test.go [hunk 1]
```diff
@@ -689,14 +689,19 @@ func TestTasksRunningLimit(t *testing.T) {
...
-				taskState, err := s.storage.GetTask(ctx, task)
+				taskState, err := s.storage.GetTask(ctx, task.ID)
 				require.NoError(t, err)
 
 				if taskState.TaskType == "long" {
 					runningLongTaskCount++
 				}
 			}
-			require.LessOrEqual(t, runningLongTaskCount, inflightLongTaskPerNodeLimit)
+
```
Comment (jkuradobery) [inline cloud/tasks/tasks_tests/tasks_test.go:699]: Do not introduce unrelated changes in the same commit. Commit shall be an atomic change which transitions from working project state to working project state.

File: cloud/tasks/storage/storage_ydb_test.go [hunk 2]
```diff
@@ -1284,14 +1292,22 @@ func TestStorageYDBListTasksCancelling(t *testing.T) {
...
+		},
+		TaskInfo{
+			ID:           taskIDStallingWhileCancelling,
+			GenerationID: generationID,
+			TaskType:     "task1",
+		},
 	}
 
-	taskIDs, err := storage.ListTasksCancelling(ctx, 100500)
+	taskInfos, err := storage.ListTasksCancelling(ctx, 100500)
```
Comment (jkuradobery) [inline cloud/tasks/storage/storage_ydb_test.go:1308]: Just tasks?

**Review by jkuradobery — APPROVED** [review]

**Review by jkuradobery — APPROVED** [review]

**Review by leftmain — APPROVED** [review]
